system/src/system_network_internal.h

```diff
@@ -387,7 +386,7 @@ class ManagedNetworkInterface : public NetworkInterface
             WLAN_CONNECTED = 0;
             WLAN_DHCP = 0;
 
-            cloud_disconnect();
+            cloud_disconnect_graceful();
```

Comment:
Yeah, I meant `WiFi.off()` and `Cellular.off()` specifically, which I think should turn off the networking immediately (at least by default). The problem with current code is that `ManagedNetworkInterface::off()` calls `ManagedNetworkInterface::disconnect()` first and thus always waits for unconfirmed messages.